### PR TITLE
Hotfix/jenkins 35820 paths

### PR DESCRIPTION
**Pull request by scherler**
Related to issue # . 

Summary of this pull request: 
.
.Fixi the paths ending with or without /
Cleanup on unMount to remove nodeId pointer
use newest JDL to fix the expanded state
.

@reviewbybees

### CODE REVIEW

**Comment by michaelneale** [comment]
🐝 passes tests

File: blueocean-dashboard/package.json [hunk 1]
```diff
@@ -35,7 +35,7 @@
     "skin-deep": "^0.16.0"
   },
   "dependencies": {
-    "@jenkins-cd/design-language": "0.0.48",
+    "@jenkins-cd/design-language": "0.0.51",
```
Comment (kzantow) [inline blueocean-dashboard/package.json]: Hooray!

File: blueocean-dashboard/src/main/js/components/PipelineRunGraph.jsx [hunk 2]
```diff
@@ -153,13 +153,17 @@ export default class PipelineRunGraph extends Component {
                         // if path ends with pipeline we simply add the node id
                         if (pathname.endsWith('pipeline/')) {
                             this.props.router.push(`${pathname}${id}`);
+                        } else if (pathname.endsWith('pipeline')) {
```
Comment (kzantow) [inline blueocean-dashboard/src/main/js/components/PipelineRunGraph.jsx]: `if(!pathname.endsWith('/')` ? :-/

**Comment by kzantow** [comment]
:bee:

**Pull request merged**